lib/plugins/acts_as_watchable/lib/acts_as_watchable.rb

```diff
@@ -54,12 +54,12 @@ module ClassMethods
         def acts_as_watchable(_options = {})
           return if included_modules.include?(Redmine::Acts::Watchable::InstanceMethods)
           class_eval do
-            has_many :watchers, as: :watchable, dependent: :delete_all
+            has_many :watchers, as: :watchable, dependent: :delete_all, validate: false
```

Comment:
This was done in a commit marked as [WIP]. Do you intend to change this back @myabc?